src/backends/ilp-quoter/index.js

```diff
@@ -42,9 +42,10 @@ class ILPQuoter {
    */
   * connect () {
     const uris = _.uniq(this.currencyPairs.map((pair) => this.backendUri + '/pair/' + pair[0] + '/' + pair[1]))
-    yield uris.map((uri) => this.putPair(uri))
-    this.backendStatus = healthStatus.statusOk
-    return
+    const results = yield uris.map((uri) => this.putPair(uri))
```

Comment:
So `results` is an array of booleans? I didn't realize you can `yield` over a map in which the callback function returns a generator.